tasks/ss-main.yml

```diff
@@ -90,6 +90,38 @@
   when: "ss_lib_dir_check.stat.isdir is defined and ss_lib_dir_check.stat.isdir"
   tags: "amsrc-ss-pydep"
 
+- name: "Create virtualenv for archivematica-storage-service"
+  pip:
+    name: "pip"
+    chdir: "{{ archivematica_src_dir }}/archivematica-storage-service"
+    virtualenv: "/usr/share/python/archivematica-storage-service"
+    state: "latest"
+  tags: "amsrc-ss-pydep"
+
+- name: "Update pip in virtualenv for archivematica-storage-service"
```

Comment:
This section is not needed, I don't think?  It duplicates the 'create virtualenv' section above, and can be removed.